components/atoms/ActionButton.js

```diff
@@ -1,27 +1,57 @@
 import PropTypes from "prop-types";
+import Link from "next/link";
 
 /**
  * Button component
  */
 export function ActionButton(props) {
-  return (
+  //Styling for buttons and links
+  const defaultStyle =
+    "bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light";
+  const secondaryStyle =
+    "bg-white text-custom-blue-blue border border-custom-blue-blue active:bg-gray-400 hover:bg-gray-200";
+  const disabledStyle = "bg-gray-light text-gray-600 border border-gray-md";
+  return props.href ? (
+    <Link href={props.href}>
+      <a
+        className={`flex justify-center content-center h-auto p-1 w-max rounded-sm py-2 px-4 focus:ring-1 focus:ring-black focus:ring-offset-2
+        ${
+          !props.secondary && !props.disabled && !props.custom
+            ? defaultStyle
+            : props.className
+        }
+        ${props.secondary && !props.disabled ? secondaryStyle : props.className}
+        ${props.custom && !props.secondary ? props.custom : ""}
+        ${
+          props.disabled ? disabledStyle : props.className
+        } text-xs md:text-base font-body`}
+        onClick={props.onClick}
+        type={props.type}
+        id={props.id}
+        data-testid={props.dataTestId}
+        data-cy={props.dataCy || props.id}
+        data-cy-button={props.dataCyButton}
+        disabled={props.disabled}
+      >
+        {props.icon ? (
+          <span className={props.icon} data-testid={props.dataTestId} />
+        ) : undefined}
+        {props.text}
+        {props.children}
+      </a>
+    </Link>
+  ) : (
     <button
       className={`flex justify-center content-center h-auto p-1 rounded-sm py-2 px-4 focus:ring-1 focus:ring-black focus:ring-offset-2
       ${
         !props.secondary && !props.disabled && !props.custom
-          ? "bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light"
-          : props.className
-      }
-      ${
-        props.secondary && !props.disabled
-          ? "bg-white text-custom-blue-blue border border-custom-blue-blue active:bg-gray-400 hover:bg-gray-200"
+          ? defaultStyle
           : props.className
       }
+      ${props.secondary && !props.disabled ? secondaryStyle : props.className}
       ${props.custom && !props.secondary ? props.custom : ""}
       ${
-        props.disabled
-          ? "bg-gray-light text-gray-600 border border-gray-md"
-          : props.className
+        props.disabled ? disabledStyle : props.className
       } text-xs md:text-base font-body`}
```

Comment:
actually one more thing since you're modifying this component. lets move these styles up before the ternary operations